### PR TITLE
Estherchee/styling

### PR DESCRIPTION
**Pull request by estherchee**
Hi, I styled the navigation bar and homepage. This is the work-in-progress version to present during the sprint review.

### CODE REVIEW

File: frontend/src/components/posts/postform.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+. postform {
```
Comment (NeuralAlchemist) [inline frontend/src/components/posts/postform.css:1]: extra space before postform selector

File: frontend/src/components/navigation/Navbar.jsx [hunk 2]
```diff
@@ -1,28 +1,41 @@
...
 
 export default function Navbar({ onLogout }) {
+  const logoObject = require('../../assets/images/logo.svg');
+  const logoURL = logoObject;
+  const homeObject = require('../../assets/images/home.svg');
+  const homeURL = homeObject;
+  const userObject = require('../../assets/images/user.svg');
+  const userURL = userObject;
+  const bellObject = require('../../assets/images/bell.svg');
+  const bellURL = bellObject;
```
Comment (NeuralAlchemist) [inline frontend/src/components/navigation/Navbar.jsx:14]: should the Object be Object.default to get the image?

File: frontend/src/components/navigation/Navbar.jsx [hunk 3]
```diff
@@ -1,28 +1,41 @@
...
     <nav>
-      <Link to="/">Logo</Link>
-
-      <div>
-        <ul>
-          <li>
-            <Link to="/posts">Home</Link>
-          </li>
+      <Link to="/posts">
+        {' '}
```
Comment (NeuralAlchemist) [inline frontend/src/components/navigation/Navbar.jsx:18]: what is the empty braces for?

**Review by NeuralAlchemist — APPROVED** [review]
Nice work on the scripts in package.json

**Review by eeels22 — APPROVED** [review]
Thanks Pei!

**Pull request merged**